runtime/oti/j9protos.h

```diff
@@ -1133,6 +1133,52 @@ extern J9_CFUNC struct J9Method* jitResolveStaticMethodRef(J9VMThread *vmStruct,
 extern J9_CFUNC struct J9Method* jitResolveSpecialMethodRef(J9VMThread *vmStruct, J9ConstantPool *constantPool, UDATA cpOrSplitIndex, UDATA resolveFlags);
 extern J9_CFUNC struct J9Class * jitGetDeclaringClassOfROMField(J9VMThread *vmStruct, J9Class *clazz, J9ROMFieldShape *romField);
 
+typedef struct J9MethodFromSignatureWalkState {
+	const char *className;
+	U_32 classNameLength;
+	struct J9JNINameAndSignature nameAndSig;
+	struct J9VMThread *vmThread;
+	struct J9ClassLoaderWalkState classLoaderWalkState;
+} J9MethodFromSignatureWalkState;
+
+/**
+ * Cleans up the iterator after matching all methods found.
+ *
+ * @param state The inlialized iterator state
+ */
+extern J9_CFUNC void
+allMethodFromSignatureEndDo(J9MethodFromSignatureWalkState *state);
```

Comment:
I suggest the prefix of this trio of functions should be `allMethods` (i.e. plural) for consistency with existing iterators (e.g. `allClassLoaders*`).

Reply:
Fixed in a36864f

Reply:
You adjusted the uses, but I don't see the corresponding change to the implementation.

Reply:
Whoops! Right I forgot to save before commit. It is there now in the latest [force-push](https://github.com/eclipse/openj9/compare/a36864fc8e2669c8781780a913ad1f798e655fa8..a7196e3b90d1786b51be17d2f9c1cc30b116ea5e) to the original commit.